CHANGES.rst

```diff
@@ -5,7 +5,7 @@ Changelog of lizard-auth-client
 1.12 (unreleased)
 -----------------
 
-- Nothing changed yet.
+- Log authentication failures at info level.
```

Comment:
Misschien nog even in één zinnetje de reden erachter zetten voor de duidelijkheid.